### ISSUE TITLE
Neorg doesn't hide formating characters, eg: like bold, italic, links, etc...

### ISSUE THREAD

**Issue by mkblast** [nvim-neorg/neorg#969]
### Prerequisites

- [X] I am using the latest stable release of Neovim
- [X] I am using the latest version of the plugin

### Neovim Version

V0.9.1

### Neorg setup

```lua
return {
  "nvim-neorg/neorg",
  build = ":Neorg sync-parsers",
  dependencies = { "nvim-lua/plenary.nvim" },
  config = function()
    require("neorg").setup {
      load = {
        ["core.defaults"] = {}, -- Loads default behaviour
        ["core.concealer"] = {
          config = {
            folds = false,
            icon_preset = "diamond"

          }
        },                  -- Adds pretty icons to your documents
        ["core.dirman"] = { -- Manages Neorg workspaces
          config = {
            workspaces = {
              notes = "~/Documents/notes",
            },
            index = "index.norg",
            default_workspace = "notes",
          },
        },
        ["core.keybinds"] = {
          config = {
            neorg_leader = ",",
          }
        },
        ["core.completion"] = {
          config = {
            engine = "nvim-cmp"
          }
        },
        ["core.presenter"] = {
          config = {
            zen_mode = "zen-mode"
          }
        },
      },
    }
  end,
}
```

### Actual behavior

According to Vhyroo from his youtube videos (I only watched the first 2 vid), Neorg should hide the formatting characters on it's own. In my setup it does not.

![2023-06-30T21:34:38,545232048+01:00](https://github.com/nvim-neorg/neorg/assets/73954836/c3327ac6-c115-4d97-8929-2d77e64ec154)

### Expected behavior

As of the video to hide the characters.

### Steps to reproduce

It's the same with the default config. So install Neorg following the github instructions.

### Help

None

**Comment by benlubas** [comment]
See `:h conceallevel` and `:h concealcursor`

This is something that really should be in the readme or conceal wiki.

**Comment by mkblast** [comment]
Many thanks. I set `vim.opt.conceallevel = 3` and it's now hidden if I change the line.

Do I close this issue or keep as a reminder to document it in the future?

**Comment by arsmoriendy** [comment]
it's already documented here https://github.com/nvim-neorg/neorg/blob/c801ae5165423f4c70841e548208c74da86acf72/doc/neorg.norg#L21

**Comment by benlubas** [comment]
I know, I'm just voting to make it more visible

**Comment by top4ek** [comment]
I have exact the same issue: links are working, but not hidden. Bold/italic/striked texts are not rendered, but ticks in tasks and stars in lists rendered correctly. Setting conceallevel doesn't help. 
I'm using alacritty with tmux and nvim-tree renders at least bold text correctly. 

![Screenshot_20230712_212904](https://github.com/nvim-neorg/neorg/assets/16016/b5953c52-464c-43b4-be8e-ce1d16711c67)

**Comment by i-interprete-on-the-fly** [comment]
I got the same issue, as the poster before me,  setting the conceallevel does not make bold/italic/striked texts concealed and also the links are working but not concealed.

**Comment by max397574** [comment]
are you sure the conceallevel isn't reset anywhere?
does it work if you run the command on the cmdline while in a neorg file? (`:set conceallevel=2`)

**Comment by top4ek** [comment]
Yep, it's(manual command) first what I tried while trying to figure out what's wrong.

**Comment by i-interprete-on-the-fly** [comment]
For me, setting the conceallevel manually does not work either.

**Comment by top4ek** [comment]
Oh, yea, sorry. :) 
level only is set once in lazy's config right before requiring neorg package. Tried setting it after or removing it at all — doesn't affect any visible things.

Only difference i see: 
When i set the level at 0 nothing is hidden at all. As I understand, it's ok.
When I set the level to 1, 2 and 3 there's only star inside a link changes to its regular ASCII look. And it's switches to it's fancy blue look when current line is a line with that link.

One strange thing I've noticed right now: * List and ** List are rendered as a colored dot despite level setting.

**Comment by i-interprete-on-the-fly** [comment]
Do we need to open another issue for this? In addition to the previously mentioned aspects, also the code block functionality does not work for me.

**Comment by top4ek** [comment]
Since some time ago, concealer began to work properly except one bug with word warp. Will write about it in common thread.
